karapuzio/assignment2/q1_ancestors/AncestorsTest.java

```diff
@@ -0,0 +1,43 @@
+package test.edu.codeU.assignment2.Ancestors;
```

Comment:
This is quite confusing to have package name where the last element looks like a class name. Can you change this to follow the java standards of package naming?
https://docs.oracle.com/javase/tutorial/java/package/namingpkgs.html